source/develop/release-management/features/storage/4k-blocks.md

```diff
@@ -0,0 +1,165 @@
+---
+title: 4k drive support
+category: feature
+authors: dchaplyg
+feature_name: 4k drive support
+feature_modules: vdsm
+feature_status: Planning
+---
+
+# 4k drive support
+
+# Summary
+
+VDSM assumes, that 512 bytes is a minimally accessible block on the storage device, which can be written or read atomically. That was true for a long time, but now most of the storage devices changed their block size to the 4096 bytes (4 kilobytes). It is still compatible with VDSM, cause while we are requesting 512 atomic read or write, underlying storage will atomically read 4096 bytes, throw away 3584 bytes and return just 512. As it still acts same way, it has no impact on the VDSM. 
+
+Unfortunately, some 4k block storage devices either refuse to read/write smaller blocks atomically (some SAN appliances) or have a performance impact while emulating 512 bytes block device (like VDO). 
```

Comment:
We discussed previously that VDO does not support atomic writes in emulation mode. Do we have new info about this or you just forgot to mention it?